cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -64,7 +64,7 @@ spec:
           name: https
           protocol: TCP
       - name: metrics-server-nanny
-        image: k8s.gcr.io/addon-resizer:1.8.7
+        image: us.gcr.io/k8s-artifacts-prod/autoscaling/addon-resizer:1.8.9
```

Comment:
Why isn't this available in global gcr.io?

Reply:
+1, we should use k8s.gcr.io, image needs to be promoted there. I'll send you instructions how to do it

Reply:
I'm interested in how to do it as well, shouldn't it happen automatically?

Reply:
@mm4tt - the instruction you sent me are outdate - they used to be valid before we deprecated the old repository

@x13n - I wasn't the one setting that up, but it seems it's not only the case for autoscaling - we seem to no longer promoting images to global gcr - you might want to follow up with people that were setting that up:
https://github.com/kubernetes/k8s.io/tree/master/k8s.gcr.io

Reply:
They are deprecated, but still applicable, as k8s.gcr.io vanity domain flip hasn't happened yet, see https://github.com/kubernetes/release/issues/270

Reply:
So why we we're even allowing for pushing images to the new repo? That means pushing to two different places - seems strange at least...

Reply:
So no matter how ridiculous it is, I updated to the k8s.gcr.io repo.